tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -48,6 +46,8 @@
 
   public static final String WORKER_JOB_NAME = "worker";
   public static final String PS_JOB_NAME = "ps";
+  public static final String NOTEBOOK_JOB_NAME = "Notebook";
+  public static final String DRIVER_JOB_NAME = "Driver";
```

Comment:
Not sure I understand what this is used for?

Reply:
This is used for sending the address:port of notebook back to TonyClient